geom/gdml/src/TGDMLWrite.cxx

```diff
@@ -1477,8 +1477,8 @@ XMLNodePointer_t TGDMLWrite::CreateElConeN(TGeoScaledShape * geoShape)
    Double_t sy = geoShape->GetScale()->GetScale()[1];
    Double_t ry1 = sy * rx1;
 
-   std::string format(TString::Format("%s/%s", fltPrecision.Data(), fltPrecision.Data()));
-   fGdmlE->NewAttr(mainN, 0, "dx", TString::Format(format.c_str(), rx1, z));
+   std::string format(TString::Format("%s/%s", fltPrecision.Data(), fltPrecision.Data()).Data());
+   fGdmlE->NewAttr(mainN, 0, "dx", TString::Format(format.c_str(), rx2, z));
```

Comment:
Is the change from `rx1` to `rx2` intentional, and if so, could you move it into a separate commit and motivate the change in the commit log?